e2e/test/scenarios/dashboard-filters/old-parameters.cy.spec.js

```diff
@@ -63,14 +68,35 @@ describe("scenarios > dashboard > OLD parameters", () => {
     it("should work", () => {
       cy.findAllByText("Doohickey");
 
-      cy.contains("Category").click();
+      cy.button("Category").click();
       popover().within(() => {
         cy.findByText("Gadget").click();
         cy.findByText("Add filter").click();
       });
 
       // verify that the filter is applied
       cy.findByText("Doohickey").should("not.exist");
+
+      // test prevent auto-apply filters in slow dashboards metabase#29267
+      openDashboardSidebar();
+
+      const dashboardParameters = cy.findByTestId(
```

Comment:
There is `filterWidget` helper for this, please search how it's used in other tests